common/djangoapps/terrain/steps.py

```diff
@@ -230,8 +230,6 @@ def i_answer_prompts_with(step, prompt):
 @step('I run ipdb')
 def run_ipdb(_step):
     """Run ipdb as step for easy debugging"""
-    import ipdb
```

Comment:
Looks like this was the sole purpose of this method, why did you remove the breakpoint?

Reply:
Return.